src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocStyleCheck.java

```diff
@@ -346,9 +349,16 @@ private void checkComment(final DetailAST ast, final TextBlock comment) {
      */
     private void checkFirstSentenceEnding(final DetailAST ast, TextBlock comment) {
         final String commentText = getCommentText(comment.getText());
+        final String[] sentences = commentText.split("\\.(?=\\s|$)");
```

Comment:
Please make this a compiled pattern also.

Reply:
Done.

Reply:
Bad naming on my part, please change this to `SENTENCE_SEPARATOR`

Reply:
Done.